shell/browser/electron_browser_main_parts.cc

```diff
@@ -280,6 +266,11 @@ void ElectronBrowserMainParts::PostEarlyInitialization() {
   // Initialize field trials.
   InitializeFieldTrials();
 
+  // Reinitialize logging now that the app has had a chance to set the app name
+  // and/or user data directory.
+  logging::InitElectronLogging(*base::CommandLine::ForCurrentProcess(),
```

Comment:
Should this be avoided for `--enable-logging --log-file` case as mentioned in your comment or is it a follow-up ?